settings.gradle

```diff
@@ -13,6 +13,10 @@ dependencyResolutionManagement {
       String keyString = key.toString()
       if (keyString.startsWith("override_")) {
         overrides.put(keyString, value.toString())
+        if (keyString.endsWith("_kotlin")) {
+          // TODO hardcoded to match what's in libs.versions.toml, but kinda ugly
```

Comment:
No need to change in this PR but in a followup could you add more detail around why we need to do this / when it will no longer be necessary? Is this to work around Kotlin 1.8 support?